### ISSUE TITLE
node-fetch - npm install vunerability

### ISSUE THREAD

**Issue by dinoluck** [justadudewhohacks/face-api.js#713]
After installing face-api.js npm returns low vulnerability that can't be fixed. Output of npm audit:

```
Low           Denial of Service                               
Package       node-fetch                                      
Patched in    >=2.6.1 <3.0.0-beta.1|| >= 3.0.0-beta.9         
Dependency of face-api.js                                     
Path          face-api.js > @tensorflow/tfjs-core > node-fetch
```

Running npm audit fix fails.

**Comment by JefferyHus** [comment]
Have you tried `npm audit fix --force`?

**Comment by dinoluck** [comment]
> Have you tried `npm audit fix --force`?

No, mostly because I saw node-fetch causing issues for other people with other packages when I was looking into it.

**Comment by JefferyHus** [comment]
This issue is related to packages depending on `node-fetch`, to not update their modules and end up with vulnerabilities. But it makes sense because they have to read the new updates of `node-fetch` in order to maintain a stable package.

I believe that using `--force` flag will get rid of the vulnerability but will create other issues with your `face-api` package.

You have two options:
- Wait `face-api` contributors to fix it
- Fix it and open a PR (win-win situation)

**Comment by vladmandic** [comment]
I've updated all packages and switched to TFJS 2.0 branch in my fork if you want to try   
<https://github.com/vladmandic/face-api>

**Comment by HaNdTriX** [comment]
If you use pnpm you can use [overrides](https://pnpm.io/9.x/package_json#pnpmoverrides) to ensure pnpm resolves to node-fetch to another version:


```json
{
  ...
  "pnpm": {
    "overrides": {
      "node-fetch": "^3.3.2"
    }
  }
}
